src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -329,11 +334,47 @@ const getCustomElementCustomTransformer = (
     style: 'static',
     styleImportData: 'queryparams',
   };
-  return [
+  const beforeCustomTransformers = [
```

Comment:
likewise possibly this array should be `customBeforeTransformers`

Reply:
https://github.com/ionic-team/stencil/pull/4501/commits/89f9ea20c7caa4d90e1a755e329e47ca10ec1803